### DISCUSSION TITLE
Difference between core_interfaces and l3_edge

### DISCUSSION THREAD

**Discussion by PieterL75** [aristanetworks/avd#4456] [FAQ]
I'm wondering what the difference between l3_edge and core_interfaces is
https://avd.arista.com/4.10/roles/eos_designs/docs/input-variables.html#l3-edge-and-dci-settings
https://avd.arista.com/4.10/roles/eos_designs/docs/input-variables.html#core-interfaces-settings
The description is the same, and (for my setup) it generated the same config...

I would expect that the l3-edge also generates the EVPN overlay, and the core_interfaces only the underlay.
But I must be misunderstanding this, so anyone who can elaborate ?

**Comment by ClausHolbechArista** [comment]
No difference. It is the same internal code handling the two models.

**Comment by PieterL75** [comment]
Interesting !
thx for the clarification
